### DISCUSSION TITLE
Error while trying to download environment on VS Code

### DISCUSSION THREAD

**Discussion by frostsxx** [microsoft/powerplatform-build-tools#720] [Q&A]
Hi community!

I was trying to download my PowerPlatform environment and I got the following error:

<img width="709" alt="Captura de ecrã 2024-01-23 160925" src="https://github.com/microsoft/powerplatform-build-tools/assets/117097511/abe310c9-7c55-40c1-90ca-38fbfeaac7c2">

I already made changes in this environment with a Power Pages project, it´not a fresh new one.

Thanks in advance!

**Comment by parvezghumra** [comment]
@frostsxx Have you verified whether the record referenced in the error message exists in the table referenced and that a value exists in the column being referenced? Is your site using the Standard Data Model or Enhanced Data Model?
